export function functionCall(){
    throw Error("error message")
}
